frontend/src/components/clinics/components/add-clinic/add-clinic.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
```

Comment:
Rename it as `add-clinic-popup` and wrap it to `modal` component
But pass the `isOpen` prop from the parent